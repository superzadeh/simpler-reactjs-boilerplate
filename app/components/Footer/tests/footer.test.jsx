import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../footer';

const child = <h1>Test</h1>;
const getElement = () => (<Footer>{child}</Footer>);

describe('Footer', () => {
  it('should render correctly', () => {
    const footer = getElement();
    expect(footer).toMatchSnapshot();
  });
  it('should render an <div> tag', () => {
    const renderedComponent = shallow(getElement());
    expect(renderedComponent.type()).toEqual('div');
  });
});
