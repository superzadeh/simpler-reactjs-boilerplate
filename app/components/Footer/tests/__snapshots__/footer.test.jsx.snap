// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<Footer>
  <h1>
    Test
  </h1>
</Footer>
`;
